lenv

```diff
@@ -165,7 +165,8 @@ sub download
 
     # verify checksum
     if( $checksum ){
-        $sum = `$OPENSSL md5 -r $src`;
+        $sum = `$OPENSSL md5 $src`;
+        $sum =~ s/^\S+\s//;
```

Comment:
thanks! i agree to your changes to improvements.
but, i thought, how about to change the matching pattern `$sum !~ /^$checksum/` to `$sum !~ /$checksum/`?